spec/requests/api/merge_requests_spec.rb

```diff
@@ -342,6 +357,12 @@
       json_response['note'].should == 'My comment'
     end
 
+    it "should return comment for merge_request by iid" do
+      post api("/projects/#{project.id}/merge_request/#{merge_request.iid}/comments", user), note: "My comment"
```

Comment:
Line is too long. [111/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.